test/e2e-v2/cases/storage/storage-cases.yaml

```diff
@@ -42,8 +42,11 @@ cases:
     expected: expected/metrics-has-value.yml
 
   # trace segment list
-  - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql trace ls  --tags http.method=POST
+  - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql trace ls --tags http.method=POST
     expected: expected/traces-list.yml
+  # negative tags search: relationship should be logical AND instead of logical OR
+  - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql trace ls --tags http.method=POST,http.status_code=200
```

Comment:
Is there no trace segment having both `http.method=POST` and `http.status_code=200`? Because of the test cases?

Reply:
`http.status_code >= 400`? This is strange. If there is a HTTP Post, it should be 200.

Reply:
> `http.status_code >= 400`? This is strange. If there is a HTTP Post, it should be 200.

Yes, here it is `200`. I mean `status_code` will be recorded as a tag only if it is `>= 400`.

For example, https://github.com/apache/skywalking-java/blob/3997f0256056788bd054ee37e4603c11c0fd6756/apm-sniffer/apm-sdk-plugin/spring-plugins/mvc-annotation-commons/src/main/java/org/apache/skywalking/apm/plugin/spring/mvc/commons/interceptor/AbstractMethodInterceptor.java#L204-L207